docs/apis/cli.md

```diff
@@ -187,6 +187,8 @@ Action "run" compiles and runs a program.
                                           java.net.URLClassLoader}.
      -d,--detached                        If present, runs the job in detached
                                           mode
+        --configDir </path/to/confdir>    The configuration directory with which
```

Comment:
This is a space, I checked. Do you want it to be aligned to the beginning of the other lines or is it ok like it is?